integration-tests/serverless/test-project/.env.yaml

```diff
@@ -1,4 +1,4 @@
 NODE_OPTIONS: "-r dd-trace/init"
-DD_TRACE_DEBUG: "true"
+DD_LOG_LEVEL: "debug"
```

Comment:
since the mini agent now inherits all stdio from the parent, the tracer doesn't need to be set to the debug level to show mini agent logs